python/tvm/relay/quantize/rewrite.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@tqchen Does it make sense to move this to newly created directory - `python/tvm/relay/op/qnn`?

This will ensure the `quantize` python namespace is not polluted.